### ISSUE TITLE
Print PDF issue

### ISSUE THREAD

**Issue by ghost** [fay59/x86doc#1]
Hi,
In the "How To Run" step3 : you said that we need to print pdf before runing extract.py script.
I used CutePDF to print (vol A & B) to PDF, but i had got error when runing extract.py.
So could you please well explain the step3 ?

Regards.

**Comment by fay59** [comment]
You are right, this is not what it should say. The problem with the raw document is that PDFMiner does not support the document's encryption type, and printing to PDF from Mac OS X solved the issue.

If that's what you're after, you can download the documentation set at [felixcloutier.com/x86/](http://felixcloutier.com/x86/).

**Comment by ghost** [comment]
Thanks for your reply.

I had already downloaded the documentation from your website.And i was trying to follow the steps to generate AVX instructions (html) from Intel Instruction Set Extensions PDF,unfortunately , i don't have a Mac.

Anyway i will keep trying.. and if i find a solution i will let you know.

Regards.

**Comment by rgosens2** [comment]
https://smallpdf.com/unlock-pdf
